examples/src/main/java/nl/knaw/dans/lib/dataverse/example/DataverseCreate.java

```diff
@@ -44,7 +44,5 @@ public static void main(String[] args) throws Exception {
         DataverseHttpResponse<Dataverse> r = client.dataverse("root").create(dataverse);
         log.info("--- END JSON OBJECT ---");
         log.info("Status Line: {}", r.getHttpResponse().getStatusLine());
-        log.info("Created: {}", r.getData().getDescription());
```

Comment:
Why have you removed these lines?

Reply:
It was because in `DataverseApi.publish()` this method is called, and if the `test` dataverse already exists, these loggings would throw an exception.

Reply:
If `test` already exists I would expect `create` to throw an exception, so I am puzzled it apparently didn't.

Reply:
Yes, it doesn't throw an exception, but returns a message:
{"status":"ERROR","message":"A dataverse with alias test already exists"}

Reply:
Ah, you're right. I messed up my boolean logic. Just pushed a fix (https://github.com/DANS-KNAW/dans-dataverse-client-lib/commit/324de2ec2cfe665611d452011909c82f8821a695). Please merge back master. This should now throw a `DataverseException`.